includes/engineCore.as

```diff
@@ -117,14 +117,14 @@ public function rawOutputText(output:String, purgeText:Boolean = false):void
 		//if (!debug) mainText.htmlText = output;
 		//trace("Purging and writing Text", output);
 		clearOutput();
-		this.output.text(output);
+		this.output.text(output,false);
```

Comment:
Replacing this with 
```as
this.output.raw(output);
```
should actually do the trick, without changing Output.as

Same fix below